pkg/plugins/transform_plugin.go

```diff
@@ -28,45 +28,38 @@ type TransformPlugin struct {
 	*TransformWrapper
 }
 
-func (tp *TransformPlugin) Load(decoder *json.Decoder, pluginDir string) error {
-	if err := decoder.Decode(&tp); err != nil {
+func (p *TransformPlugin) Load(decoder *json.Decoder, pluginDir string) error {
+	if err := decoder.Decode(&p); err != nil {
```

Comment:
I think we don't need a pointer to a pointer here? I tested the below and it compiles at least:
```suggestion
	if err := decoder.Decode(p); err != nil {
```

I know it's not directly related to the changes here, but we might as well clean a bit.